packages/app-headless-cms/src/admin/components/ContentModelForm/ContentModelFormRender/fields/RichTextInput.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { I18NValue } from "@webiny/app-i18n/components";
+import I18NRichTextEditor from "@webiny/app-i18n/admin/components/I18NRichTextEditor";
+import { CmsContentModelModelField } from "@webiny/app-headless-cms/types";
+import { BindComponentRenderProp } from "@webiny/form";
+import numberedListPlugins from "@webiny/app-i18n/admin/plugins/richTextEditor/numberedList";
+import bulletedListPlugins from "@webiny/app-i18n/admin/plugins/richTextEditor/bulletedList";
+import headingPlugins from "@webiny/app-i18n/admin/plugins/richTextEditor/heading";
+import codePlugins from "@webiny/app-i18n/admin/plugins/richTextEditor/code";
+
+type Props = {
+    bind: BindComponentRenderProp;
+    field: CmsContentModelModelField;
+};
+
+const richTextPlugins = [
```

Comment:
Actually, I thought we will move all of these plugins into the rich editor directly.
`packages/app-i18n/src/admin/plugins/richTextEditor/index.ts`
`packages/app-i18n/src/site/plugins/richTextEditor/index.ts`

I think it'll be a nice addition for the I18N app's rich text editor as well.

Reply:
We can do this post-beta launch if it turns out it's complicated. Just create a new issue if so.